### PR TITLE
Fix virginity cost errors

### PR DESCRIPTION
**Pull request by GynoidDoesGames**
Fixes slavery value for characters w/ retained virginity. Also adds an underflow check in there for good measure.

@Innoxia

### CODE REVIEW

File: src/com/lilithsthrone/game/character/GameCharacter.java [hunk 1]
```diff
@@ -3653,6 +3653,11 @@ public int getValueAsSlave(boolean includeInventory) {
 			}
 		}
 		
+		if(value < 0)
+		{
+			value = 0;
+		}
+		
 		return value;
```
Comment (Pimvgd) [inline src/com/lilithsthrone/game/character/GameCharacter.java]: return Math.max(0, value);

Comment (GearheadLydia) [reply src/com/lilithsthrone/game/character/GameCharacter.java]: Thanks.

Comment (GynoidDoesGames) [reply src/com/lilithsthrone/game/character/GameCharacter.java]: Thanks.

**Comment by Innoxia** [comment]
Thank you!

**Pull request merged**